hail_search/queries/base.py

```diff
@@ -278,39 +278,48 @@ def _parse_sample_data(self, sample_data):
         logger.info(f'Loading {self.DATA_TYPE} data for {num_families} families in {len(project_samples)} projects')
         return project_samples, num_families
 
-    def _load_filtered_project_hts(self, project_samples, skip_all_missing=False, **kwargs):
+    def _load_filtered_project_hts(self, project_samples, skip_all_missing=False, n_partitions=MAX_PARTITIONS, **kwargs):
+        if len(project_samples) == 1:
+            project_guid = list(project_samples.keys())[0]
+            project_ht = self._read_table(f'projects/{project_guid}.ht', use_ssd_dir=True)
+            return self._filter_entries_table(project_ht, project_samples[project_guid], **kwargs)
+
+        # Need to chunk tables or else evaluating table globals throws LineTooLong exception
+        # However, minimizing number of chunks minimizes number of aggregations/ evals and improves performance
+        # Adapted from https://discuss.hail.is/t/importing-many-sample-specific-vcfs/2002/8
+        chunk_size = 64
```

Comment:
Why did you choose a chunk size of 64 and 12 max partitions?

Reply:
chunk size of 64 was recommended in the linked hail discussion, and I did some testing with larger sizes and they did not work. 12 was determined experimentally and the ticket where the testing was done is linked in the comments: https://github.com/broadinstitute/seqr/blob/master/hail_search/queries/base.py#L22

Reply:
nice, thank you